cupy/core/core.pyx

```diff
@@ -439,7 +439,7 @@ cdef class ndarray:
         if strides.size() == shape.size():
             newarray = self.view()
         else:
-            newarray = self.copy()
+            newarray = ascontiguousarray(self)
```

Comment:
Do we leave these `ascontiguousarray` changes? The original `self.copy()`, whose default order is `C`, is enough as well as is natural for the intension.